src/App.scss

```diff
@@ -0,0 +1,85 @@
+$primary: blue;
+
+.App {
```

Comment:
Styled components should belong to a separate file, in general ending in `.style.js`